src/config.rs

```diff
@@ -166,6 +167,26 @@ impl IndexMut<usize> for ColorList {
     }
 }
 
+#[derive(Debug, Deserialize)]
+pub struct Shell<'a> {
+    program: Cow<'a, str>,
+    args: Vec<String>,
+}
+
+impl<'a> Shell<'a> {
+    pub fn new(program: &'a str) -> Shell<'a> {
```

Comment:
Actually, I already have a working commit with `-e` implemented. But I potentially have to clean it up before submitting a pull request. https://github.com/NickeZ/alacritty/tree/add-cli-e